_posts/en/newsletters/2019-07-24-newsletter.md

```diff
@@ -0,0 +1,149 @@
+  testnet users and anyone else interested in signet to review the
+  above code and documentation to ensure signet will fulfill your needs.
+
+- **Additional Just-In-Time (JIT) LN routing discussion:** in the JIT
+  discussion described in the [newsletter][Newsletter #54] two weeks ago,
+  contributor ZmnSCPxj explained why routing nodes needed zero-fee
+  rebalance operations in many cases for JIT routing to be incentive
+  compatible.  This week he [posted][jit with fee] a suggestion to the C-Lightning
+  mailing list on how nodes could be less incentive compatible but still
+  prevent abuse when performing paid rebalancing for JIT routing.
+  Nodes would keep track of how much routing fee they had earned from each
+  channel and spend up to that amount on rebalancing.  This would ensure
+  that a dishonest channel counterparty couldn't steal any more than it
+  had already allowed its honest peers to claim in routing fees.
+
+## Bech32 sending support
+
+*Week 19 of 24 in a [series][bech32 series] about allowing the people
+you pay to access all of segwit's benefits.*
+
+{% comment %}<!-- weekly reminder for harding: check Bech32 Adoption
+wiki page for changes -->{% endcomment %}
+
+{% include specials/bech32/19-real-fees.md %}
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[LND][lnd repo], [C-Lightning][c-lightning repo], [Eclair][eclair repo],
+[libsecp256k1][libsecp256k1 repo], [Bitcoin Improvement Proposals
+(BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #15891][] changes the node defaults when using
+  regtest mode to enforce the same standard transaction rules used on
+  mainnet.  Those rules determine which transactions nodes relay and
+  accept into the mempool.  This change should make it easier for
+  developers to test their custom transactions against the default
+  policy.  Anyone who needs the old behavior of relaying and accepting
+  any consensus-valid transaction may use the `acceptnonstdtxn`
+  configuration parameter.
+
+- [Bitcoin Core #16152][] disables [BIP37][] bloom filter support by
+  default.  This feature allowed lightweight wallets to create a [bloom
+  filter][] from a list of their addresses, send that filter to a node,
+  ask the node to scan historic blocks or new incoming transactions, and
+  receive back only those transactions that matched the filter.  This
+  allowed the lightweight client to only receive the transactions it was
+  interested in (plus maybe a few extra false-positive matches),
+  reducing its bandwidth requirements.  However, it also meant that
+  clients could require full nodes to scan and rescan through the entire
+  historic block chain over and over at no cost to the client---creating
+  the vector for a DoS attack.
+
+    For that reason and others (including [privacy concerns][filter
+    privacy]) several Bitcoin Core contributors have wanted to disable
```

Comment:
Perhaps s/several/a number of/ because several twice.

Reply:
Replaced first instance with "a number of"